tests/Integration/Infrastructure/Controller/Regulation/Fragments/UpdateMeasureControllerTest.php

```diff
@@ -93,8 +93,8 @@ public function testAddAndRemoveLocation(): void
 
         $crawler = $client->request($form->getMethod(), $form->getUri(), $values);
 
-        $this->assertResponseStatusCodeSame(303);
-        $crawler = $client->followRedirect();
+        $this->assertResponseStatusCodeSame(200);
+        /*  $crawler = $client->followRedirect(); */
         $this->assertRouteSame('fragment_regulations_measure', ['uuid' => MeasureFixture::UUID_TYPICAL]);
```

Comment:
Ici aussi il faut modifier le test pour inspecter les turbo-streams présents dans la nouvelle réponse

On peut aussi tester le nouveau contenu du bloc history

```suggestion
        $this->assertResponseStatusCodeSame(200);
        $streams = $crawler->filter('turbo-stream')->extract(['action', 'target']);

        $this->assertEquals([
            ['replace', 'block_measure_' . MeasureFixture::UUID_TYPICAL],
            ['replace', 'block_regulation_order_history'],
        ], $streams);

        $this->assertSame('Modifié le 09/06/2023', $crawler->filter('[data-testid="history"]')->text());

```